scm/etc/gmtb_scm_qsub_example.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+"""
+-----------------------------------------------------------------------
+ Description:  Example script to submit a job through the HPC batch system
+
+ Assumptions: For use on Cheyenne. This script must be copied to the bin directory.
+              The user should edit the JOB_NAME, ACCOUNT, etc.
+              ./scm/etc/Cheyenne_setup_intel.sh or
+              ./scm/etc/Cheyenne_setup_gnu.sh must be run before submitting this script
+
+ Command line arguments: none
+
+ Usage: ./gmtb_scm_qsub_example.py   # from the bin directory
+-----------------------------------------------------------------------
+"""
+
+from subprocess import Popen, PIPE
+import os
+import time
+
+USER = os.getenv('USER')
+STRINGS = [USER, 'ucar.edu']
+MY_EMAIL = '@'.join(STRINGS)
+PROC = Popen('qsub', shell=True, stdin=PIPE, stdout=PIPE, stderr=PIPE, close_fds=True)
+
+### Begin User-editable section ###
+JOB_NAME = "test_job"
+ACCOUNT = "p48503002"
+WALLTIME = "walltime=00:20:00"
+PROCESSORS = "select=1:mpiprocs=24:ncpus=24"
```

Comment:
Since the SCM is serial, do we really need to ask for 24 cpus? It is supposed to be compilable with openmp, although I haven't tested that in quite a while.

Reply:
I can change this to "select=1:ncpus=1"

Reply:
At some point, it would be nice to use multiprocessing with the multi_run script to dispatch N serial jobs to N CPUs, but that doesn't work now, so maybe just asking for 1 would be sufficient. Is this qsub example script supposed to turn into the RT script in later commits, or is this really just an example?

Reply:
In this case, will this job run in the "share" queue on cheyenne, rather than "regular"?  cheyenne charges for a full node (and exclusive use of that node) in regular, but charges per processor in share.  Later, submitting multiple executables in parallel might make sense, then a full node might make sense again.  But, then again, I doubt this job is taking much in the way of CPU hours in any form!

Reply:
> @grantfirl Pylint is a tool that will analyze your python code without actually running it. It checks for errors and enforces a coding/styling standard defined in .pylintrc. Unless you want to use it, it won't affect you. It just makes for cleaner, more consistent code. I put it in this repository so anyone that wants to use it will be using the same coding standard.
> 
> I used the multi_run script to get started running all of the supported cases/suites. I agree that:
> 
> 1. We should set up a file to hold all of the runs for the RT and modified namelists.  For example, we don't need to run the twpice cases for 2138400 sec.  I need to look into the -f option.  I'm still learning...
> 2. The multi_run script needs some sort of pass/fail criteria after each case and maybe at the end.
> 3. We will want to store baselines somewhere and do a cmp comparison.  This won't be too hard, but we need to figure out what to run for the RTs.

That sounds great about pylint. I'll want to run it on all of the python scripts in the repo at some point to be consistent.  I'm sure it will find a bunch of stuff wrong!

Starting from the supported cases/suites is a good option, even though, like you said, maybe running multi-day tests are not necessary. If we want to guarantee that the case runs to completion (and will work for users), I don't see a huge problem leaving the TWPICE case as-is though for the RT. It doesn't take that long or use much processing even at multiple days. It could also be possible with trigger-based physics options that some trigger doesn't get pulled until later in the simulation, so one would want to run the whole thing in that case too. I guess it depends on the purpose of the RT. One could maybe even define more than one type -- one to make sure it passes ccpp_prebuild and compiles and runs some subset of cases, and one that checks results for any changes. The more "mechanical" one could be run more often than the one that checks for output changes? Just a thought.